oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/BlobThroughPutTest.java

```diff
@@ -59,13 +60,11 @@ public class BlobThroughPutTest {
 
     static {
         Map<WriteConcern,String> bimap = new DualHashBidiMap<WriteConcern,String>();
-        bimap.put(WriteConcern.FSYNC_SAFE,"FSYNC_SAFE");
-        bimap.put(WriteConcern.JOURNAL_SAFE,"JOURNAL_SAFE");
+        bimap.put(WriteConcern.JOURNALED,"JOURNALED");
 //        bimap.put(WriteConcern.MAJORITY,"MAJORITY");
         bimap.put(WriteConcern.UNACKNOWLEDGED,"UNACKNOWLEDGED");
-        bimap.put(WriteConcern.NORMAL,"NORMAL");
 //        bimap.put(WriteConcern.REPLICAS_SAFE,"REPLICAS_SAFE");
-        bimap.put(WriteConcern.SAFE,"SAFE");
+        bimap.put(WriteConcern.ACKNOWLEDGED,"ACKNOWLEDGED");
```

Comment:
Those WriteConcerns are removed:

- FSYNC_SAFE and JOURNAL_SAFE replaced by JOURNALED
- NORMAL and SAFE replaced by ACKNOWLEDGED